cmd/substreams/verify.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
Remove this file, we are going to keep only the `substreams registry publish` version.